tests/CalendarTests.cpp

```diff
@@ -0,0 +1,68 @@
+#include <gtest/gtest.h>
+#include <Calender.hpp>
+#include <algorithm>
+
+struct CalendarTests : public ::testing::Test
+{
+    CalendarTests()
+    {
+        calender_.addLesson(User::Group::weekend, "2019-06-15", "16:30", "Environment configuration" );
+        calender_.addLesson(User::Group::evening, "2019-07-06", "16:30", "Programming tools" );
+        calender_.addLesson(User::Group::evening, "2019-07-07", "16:30", "Basics of C++" );
+    }
+
+    Calender calender_;
+};
+
+TEST_F(CalendarTests, can_add_and_view_Lesson)
+{
+    const auto& result = calender_.viewLesson(User::Group::evening, "2019-07-07", "16:30");
```

Comment:
what will happen if I provide date and time that does not exist for the selected group? It also should be tested.

Reply:
It will throw exception.

Test will be implemented today.